realsense/hand/js/hand_api.js

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+var HandModule = function(objectId) {
+  common.BindingObject.call(this, objectId ? objectId : common.getUniqueId());
+
+  this._registerLifecycleTracker();
+
+  if (objectId == undefined)
+    internal.postMessage('handModuleConstructor', [this._id]);
+
+  function wrapImageReturns(data) {
+    const bytesPerInt32 = 4;
+    var headerByteOffset = 4 * bytesPerInt32;
+    var int32View = new Int32Array(data, 0, 4);
+    // int32View[0] is the callback id.
+    var width = int32View[2];
+    var height = int32View[3];
+    var format, buffer;
+    if (int32View[1] === 0x00020000) {
```

Comment:
They are come from native enum value. I will add comments about its origin.

I wish that I can use protobuf someday in this project. :)